pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java

```diff
@@ -340,16 +340,11 @@ private boolean isLastMethodName(final ASTMethodCallExpression methodNode, final
     }
 
     private boolean isWithSecurityEnforced(final AbstractApexNode<?> node) {
-//        if (node instanceof ASTSoqlExpression) {
-//            String pattern = "(?i).*[^']\\s*WITH SECURITY_ENFORCED\\s*[^']*";
-//            String query = ((ASTSoqlExpression) node).getQuery();
-//            return query.matches(pattern);
-//        }
-//        return false;
-          if (node instanceof ASTSoqlExpression) {
-            return WITH_SECURITY_ENFORCED.matcher(((ASTSoqlExpression) node).getQuery()).matches();
-          }
-          return false;
+        if (node instanceof ASTSoqlExpression) {
+            boolean temp = WITH_SECURITY_ENFORCED.matcher(((ASTSoqlExpression) node).getQuery()).matches();
+            return temp;//WITH_SECURITY_ENFORCED.matcher(((ASTSoqlExpression) node).getQuery()).matches();
```

Comment:
@jarquile 
The build is still failing (see https://travis-ci.org/github/pmd/pmd/builds/661704547?utm_source=github_status&utm_medium=notification):

```
[INFO] --- maven-pmd-plugin:3.13.0:check (default) @ pmd-apex ---

[INFO] PMD version: 6.22.0

[INFO] PMD Failure: net.sourceforge.pmd.lang.apex.rule.security.ApexCRUDViolationRule:345 Rule:UnnecessaryLocalBeforeReturn Priority:1 Consider simply returning the value vs storing it in local variable 'temp'.
```

Just write `return WITH_SECURITY_ENFORCED.matcher(((ASTSoqlExpression) node).getQuery()).matches();`
And remove the comment.

Reply:
I'll fix it myself....